source/panel_finder/panel_classifier/inference_opencv.py

```diff
@@ -0,0 +1,68 @@
+from source.instance import get_json_from_path
+from source.common.module import Module
+from pathlib import Path
+import numpy as np
+import cv2
+import os
+
+
+def find_ratio(a, b):
+    """
+    Compares two numbers by finding the ratio between them. The result is between 0 and 1.
+    When using this function, please the numbers passed have the same sign.
+
+    1 means they are identical
+    0 means one is infinitely larger than the other
+    :param a: first number to compare
+    :param b: second number to compare
+    :return: ratio between them, between 0 and 1
+    """
+    if a == 0 or b == 0:  # prevent div by 0
+        return 0
+    return a / b if a < b else b / a
+
+
+class Inference(Module):
+    def __init__(self):
+        self.working_dir = Path(os.path.dirname(os.path.abspath(__file__)))
+        super().__init__(self.working_dir)
+
+    def load_model(self, path=None):
+        """
+        Loads the saved_model from path
+        :param path: path to the tensorflow frozen pb path
+        :return: The loaded model
+        """
+        if path is None:
+            model = self.properties["load_tf_model"]
+
+        return cv2.dnn.readNetFromTensorflow(str(self.working_dir / model))
+
+    @staticmethod
+    def model_predict(net, input):
```

Comment:
The parameter `input` should be renamed, as it shadows the builtin function `input()`

Reply:
changed to nn_input